circuits/sybil/sybilResCredentialAtomicQuerySigV2.circom

```diff
@@ -0,0 +1,12 @@
+pragma circom 2.0.0;
+
+include "sybilResCredentialAtomicQuerySigOffChain.circom";
+
+component main{public [
+                        // uniqueness claim
```

Comment:
Any use of this comment for a reader not in a context?

Reply:
Removed